upgrade/network/chn_enable.md

```diff
@@ -0,0 +1,856 @@
+      --customer-highspeed-network <CHN VLAN> <CHN IPv4 Subnet> \
+      --number-of-chn-edge-switches <number of edge switches> \
+      --sls-output-file ${UPDATEDIR}/sls_file_with_chn.json
+   ```
+
+   where:
+
+      - `<CHN VLAN>` is the "stub" `VLAN` for the `CHN`.  This is currently used only on the edge switches in access mode, not a trunk through the high speed network.
+      - `<CHN IPv4 Subnet>` is the pre-requisite site-routable IPv4 subnet for the `CHN`.
+      - `<number of edge switches>` is typically 2 Arista or Aruba switches, but some pre-production systems have 1.
+
+4. (`ncn-m001#`) Upload data to `SLS`.
+
+   ```bash
+   curl --fail -H "Authorization: Bearer ${TOKEN}" -k -L -X POST 'https://api-gw-service-nmn.local/apis/sls/v1/loadstate' -F "sls_dump=@${UPDATEDIR}/sls_file_with_chn.json"
+   ```
+
+### Update customizations
+
+1. (`ncn-m001#`) Move to the update directory.
+
+   ```bash
+   cd ${UPDATEDIR}
+   ```
+
+2. (`ncn-m001#`) Set the directory location for the customizations script to add `CHN`.
+
+   ```bash
+   export CUSTOMIZATIONS_SCRIPT_DIR=/usr/share/doc/csm/upgrade/scripts/upgrade/util
+   ```
+
+3. (`ncn-m001#`) Create updated `customizations.yaml` against updated `SLS`.
+
+   ```bash
+   ${CUSTOMIZATIONS_SCRIPT_DIR}/update-customizations.sh ${BACKUPDIR}/customizations.yaml > ${UPDATEDIR}/customizations.yaml
+   yq validate ${UPDATEDIR}/customizations.yaml
+   ```
+
+   **Important** If the updated `customizations.yaml` file is empty or not valid `YAML`, do not proceed.  Debug in place.
+
+4. (`ncn-m001#`) Upload new `customizations.yaml` to ensure changes persist across updates
+
+   ```bash
+   kubectl delete secret -n loftsman site-init
+   kubectl create secret -n loftsman generic site-init --from-file=${UPDATEDIR}/customizations.yaml
+   ```
+
+### Update CSM Service endpoint data (MetalLB)
+
+1. (`ncn-m001#`) Create new MetalLB configuration map from updated customizations data.  The new configmap will not be applied in the update phase as this would change service endpoints earlier than desired.
+
+   ```bash
+    yq r ${UPDATEDIR}/customizations.yaml 'spec.network.metallb' |
+    yq p - 'data.config.' |
+    sed 's/config\:/config\:\ \|/' |
+    yq m - ${BACKUPDIR}/metallb.yaml > ${UPDATEDIR}/metallb.yaml
+   ```
+
+## Migrate Phase
+
+### Migrate NCN workers
+
+1. (`ncn-m001#`) Change to updates directory
+
+   ```bash
+   cd ${UPDATEDIR}
+   ```
+
+2. (`ncn-m001#`) Ensure SHS is active by testing if there is an `HSN` IP address (typically `10.253`) on the `hsn0` interfaces.  If there is not primary address on the `hsn0` interface this must be fixed before proceeding;
+
+   ```bash
+   pdsh -w ncn-w[$(printf "%03d-%03d" 1 $(egrep 'ncn-w...\.nmn' /etc/hosts | wc -l))] ip address list dev hsn0
+   ```
+
+   **NOTE** If some interfaces have an `HSN` address and others do not, this is typically indicative that  CFS plays are failing at some point.
```

Comment:
In the Papaya recipe CFS is no longer responsible for configuring the HSN interfaces, the SHS software is baked into the NCN image and the `cray-ifscan` systemd service performs configuration of the HSN interfaces.

I wonder if there is any SHS/Slingshot troubleshooting documentation for this that we can point users to?